utils/misc.py

```diff
@@ -151,3 +151,37 @@ def download_ampere_imagenet():
 
     os.environ["IMAGENET_IMG_PATH"] = str(dataset)
     os.environ["IMAGENET_LABELS_PATH"] = str(labels)
+
+
+def download_coco_dataset():
+    from utils.downloads.utils import get_downloads_path
+    if "COCO_IMG_PATH" not in os.environ and "COCO_ANNO_PATH" not in os.environ:
+        labels_link = "http://images.cocodataset.org/annotations/annotations_trainval2014.zip"
+        images_link = "http://images.cocodataset.org/zips/val2014.zip"
+        coco_data = pathlib.Path(get_downloads_path(), "coco")
+
+        if not pathlib.Path(coco_data).is_dir():
+            try:
+                subprocess.run(["wget", labels_link])
+                subprocess.run(["wget", images_link])
+                subprocess.run(["mkdir", coco_data])
+                subprocess.run(["unzip", 'annotations_trainval2014.zip', '-d', coco_data])
+                subprocess.run(["unzip", 'val2014.zip', '-d', coco_data])
+            except KeyboardInterrupt:
+                subprocess.run(["rm", 'val2014.zip'])
+                subprocess.run(["rm", 'annotations_trainval2014.zip'])
+        else:
+            pass
+
+        dataset = pathlib.Path(coco_data, 'val2014')
+        labels = pathlib.Path(coco_data, 'annotations', 'instances_val2014.json')
+
+        os.environ["COCO_IMG_PATH"] = str(dataset)
+        os.environ["COCO_ANNO_PATH"] = str(labels)
+
+    else:
+        if "COCO_IMG_PATH" not in os.environ:
+            print("COCO_IMG_PATH variable must be set along with COCO_ANNO_PATH!")
```

Comment:
this is already handled elsewhere (in COCO utils file), not needed

Reply:
fair enough, I though your previous comment regarding this matter meant to remind the user that he set the one but not the other, but in this case this whole else statement will be removed and handled solely by coco.py